src/scripts/modules/countdown-timer.js

```diff
@@ -0,0 +1,106 @@
+const { benenson_data: benensonData } = window;
+
+const init = () => {
+  Array.from(document.querySelectorAll('.countdownTimer'))
+    .forEach((timer) => {
+      // return if timer has already finished
+      if (timer.classList.contains('is-finished')) {
+        return;
+      }
+
+      let finished = false;
+      let days = 0;
+      let hours = 0;
+      let minutes = 0;
+      let seconds = 0;
+
+      const daysElement = timer.querySelector('.countdownTimer-days span');
+      const hoursElement = timer.querySelector('.countdownTimer-hours span');
+      const minutesElement = timer.querySelector('.countdownTimer-mins span');
+      const secondsElement = timer.querySelector('.countdownTimer-secs span');
+      const endDate = new Date(timer.dataset.date).getTime();
+      const timerRef = timer.dataset.ref;
+
+      if (typeof endDate !== 'number') {
+        return;
+      }
+
+      const calculate = () => {
+        let startDate = new Date();
+        startDate = startDate.getTime();
+
+        // turn milliseconds into seconds to make calculations easier
+        let timeRemaining = parseInt((endDate - startDate) / 1000, 10);
+
+        if (timeRemaining < 0) {
+          finished = true;
+          return;
+        }
+
+        // divide time remaining by number of seconds in a day & remove days from timeRemaining
+        days = parseInt(timeRemaining / 86400, 10);
+        timeRemaining %= 86400;
+
+        // divide time remaining by number of seconds in an hour & remove hours from timeRemaining
+        hours = parseInt(timeRemaining / 3600, 10);
+        timeRemaining %= 3600;
+
+        /* divide time remaining by number of seconds in a minute &
```

Comment:
Comment styles are inconsistent here. Not really an issue, but there's a bit of cognitive overhead.